capstone/src/main/java/com/google/sps/AssociationResult.java

```diff
@@ -42,6 +47,7 @@ public AssociationResult(String content, float score, float weight) {
     this.content = content;
     this.score = score;
     this.weight = weight;
+    strongSentiment = Math.abs(score) > SENTIMENT_THRESHOLD;
     key = Optional.empty();
```

Comment:
Same as jasper's comment but with `this.key`

Reply:
Fixed!